docs/reference/plugins.md

```diff
@@ -1 +1,16 @@
 # Plugins
+
+Shift supports the integration of custom Plugins. Shift Plugins are Qt based UIs which can interact with Shift to display additional information, perform actions or modify the behaviour of the application.
+
+## Shift Native Plugins
+
+Shift ships with two example plugins which can be found in the *Tools* section in the top menu bar. The desired plugin can be opened by clicking on the corresponding entry.
+Once a Plugin is opened, the Shift UI can be customized with the best suited layout.
+
+The *Operators List* Plugin helps us visualize the list of operators in the workflow and shows the their current progress.
+
+![Opening a Plugin](/images/opening_a_plugin.gif)
+
+The *Variables Browser* allows users to parse the workflow variables currently set and check their content. This is especially helpful in large workflows where it is easy to lose track of the nodes previously executed.

```

Comment:
We should add a link to the Variables page in variables.
Question, is it possible to delete variables with this plugin too? I'm not sure if that functionality was added at the end or not.

Reply:
yes it is possible with a right click context menu.
I just added a comment about it.

Reply:
Has the link to the variables page been added? I don't see it in the paragraph

Reply:
Where should the "Variables Page" be? I only see a "Variable Node" page, are you referring to this one?

Reply:
Yes